src/components/SearchView.tsx

```diff
@@ -0,0 +1,103 @@
+import { IonList, IonCard, IonCardTitle, IonCardSubtitle } from '@ionic/react';
+import React from 'react';
+import { PageViewMode } from '../utils/SongUtils';
+import './Components.css';
+
+
+interface SearchViewProps
+{
+  searchString: any
+  setSongNumber: (_: number) => void
+  setPageViewMode: (_: PageViewMode) => void
+}
+
+interface Song
+{
+  title: string,
+  author: string,
+  songNumber: number
+}
+
+/**
+ * Search View.
+ */
+const SearchView: React.FC<SearchViewProps> = (props) => {
+  
+  let songs;
+
+  try 
+  {
+    songs = require("../resources/Songs_&_Hymns_Of_Life/BlackBookSongList.json");
+    console.log(songs.songs);
+  }
+  catch (e)
+  {
+    console.log(e);
+    throw e;
```

Comment:
Might be better to return a "Could not load songs" message, or an empty-list instead of `throw` ing again. A thrown exception like this might just crash the app. 

But I see the catch is kind of just for logging so if we think it's stable I dont mind getting rid of the `try-catch` altogether